src/browser.js

```diff
@@ -1,4 +1,13 @@
+var Client = require('./client');
+var Server = require('./server');
+var serializer = require("./transforms/serializer");
+
 module.exports = {
-  Client: require('./client'),
+  Client: Client,
+  Server: Server,
+  states: require("./states"),
+  createSerializer:serializer.createSerializer,
+  createDeserializer:serializer.createDeserializer,
+  ping: require('./ping'),
```

Comment:
.connect is defined there https://github.com/PrismarineJS/node-minecraft-protocol/blob/master/src/createClient.js#L15